remotetracker/jira.go

```diff
@@ -0,0 +1,29 @@
+package remotetracker
+
+import (
+	"github.com/andygrunwald/go-jira"
+)
+
+// Jira represents Jira remote issue tracker
+type Jira struct {
+	items []Item
```

Comment:
Why are we 'Fetching' N number of RemoteWorkItems into memory... ? What happened to the previous design?